### PR TITLE
Dangerzone video embed

### PR DESCRIPTION
**Pull request by Jwan622**
-embedded youtube video on homepage
-created a helper module that takes in a youtube url string and parses it.

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -4,3 +4,9 @@
   <%= link_to "Learn More", learn_more_path %>
   <%= link_to "Sign Up", signup_path %>
 </div>
+
+<div class="map"></div>
+
+<div class="embed-container">
```
Comment (cluhring) [inline app/views/static_pages/home.html.erb:10]: Looks good to me.  Nice job, Jeff.

**Comment by cluhring** [comment]
:space_invader:

File: app/views/static_pages/home.html.erb (hunk 1 above)
Comment (Jwan622) [inline app/views/static_pages/home.html.erb:10]: Thanks!

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -4,3 +4,9 @@
...
+  <%= embed("https://www.youtube.com/watch?v=ZzQ3eBerHfM") %>
```
Comment (bhargavisatpathy) [inline app/views/static_pages/home.html.erb:11]: Thanks for explaining the how the embed here is working.

**Pull request merged**